src/templates/common/contentFooter/index.tsx

```diff
@@ -77,6 +77,7 @@ export function ContentFooter({
           <va-button
             label="Give feedback"
             id="mdFormButton"
+            disable-analytics
```

Comment:
🤔 Why disable analytics on this button?

Reply:
This is the standard practice when we are doing custom analytics on a button click. So the `survey-show-form-call` is the replacement for this default event.

Reply:
Weirdly enough - we are missing a standard `event.cta-button-click` that exists on Prod but not on next build. I'm not convinced this matters now though because we have everything else. Let me track this down for a few mins

Reply:
@brianseek I think this is fine to leave disabled, like you said it's standard practice because it tends to prevent duplicate events being fired. If we find out later that it should be enable for whatever reason we can always change it

Reply:
if it exists on prod, that just means they are implementing that incorrectly